### ISSUE TITLE
[Request]: Queuing operations such as dependency downloads

### ISSUE THREAD

**Issue by JohannesLotterDev** [bottlesdevs/Bottles#3397]
### Tell us the problem or your need

Hi, often when trying to debug running certain apps under bottles I end up installing a lot of dependencies. The amount of time it takes to install them is not an issue for me but what is extremely frustrating is not being able to install several dependencies at the same time. It would be extremely convenient to be able to queue dependency installs or other operations so you can go do something else while the install is occurring. Instead of having to open bottles every couple minutes to check if one operation is complete so you can do the next.

### Describe the solution you'd like

Add queuing functionality so you can do several operations in succession with ease. For example queuing a "allfonts" dependency download and "dotnet" dependency download.

**Comment by cleanerspam** [comment]
Yes this would be a good feature along with an option to install all known dependency  at once would be a great time saver .On slow internet connection it takes forever to install each of them individually  . A download all option would atleast give freedom to do other task while Bottle downloads all dependecy

**Comment by dginovker** [comment]
Seems similar to #2663
